assets/components/src/with-wizard/style.scss

```
@@ -1,4 +1,64 @@
+@import '../../../shared/scss/muriel-component';
+
 .newspack-wizard .newspack-logo {
 	display: block;
 	margin: 2em auto 0 auto;
 }
+
+.muriel-wizardScreen__loading {
+	.muriel-wizardScreen {
+		position: relative;
+
+		&:after {
+			display: block;
+			content: '';
+			position: absolute;
+			top: 0;
+			left: 0;
+			right: 0;
+			bottom: 0;
+			background: $muriel-white;
+			opacity: 0.8;
+			z-index: 102; /* Toggle arrows from Select components are at z-index: 101 */
+		}
+
+		&:before {
+			display: block;
+			position: absolute;
+			content: "";
+			z-index: 103;
+			left: 0;
+			top: 0;
+			width: 10%;
+			height: 4px;
+			background-color: $primary-color;
+			animation: loading 2s linear infinite;
+		}
+
+		&.muriel-wizardScreen__no-background {
+			&:after {
+				background: #f1f1f1; /* WP admin bg color */
```

How about transparent to be safe?

```suggestion
				background: transparent;
```

---

I want it to "fade out" the stuff behind the BG, so it needs a real BG.